modules/signing_worker/files/requirements.dep.txt

```diff
@@ -41,129 +41,178 @@ aiohttp==3.7.4.post0 \
+    --hash=sha256:8dd4ac612fd60cb277eaf6e2ce02f68dda54aba06870ca6fa2b28369bf39aa14
+    # via -r https://raw.githubusercontent.com/mozilla-releng/scriptworker/cd731fa4b02b10b430f9414df678f47640491931/requirements.txt
+idna==3.2 \
+    --hash=sha256:14475042e284991034cb48e06f6851428fb14c4dc953acd9be9a5e95c7b6dd7a \
+    --hash=sha256:467fbad99067910785144ce333826c71fb0e63a425657295239737f7ecd125f3
+    # via
+    #   requests
+    #   yarl
+immutabledict==2.2.0 \
+    --hash=sha256:1f95dc5595f4f857a447d9066f361d70200820cb4352a7ad4aaf48ecdc8836d4 \
+    --hash=sha256:ed65fc95ed84413ea3bd8dd94c3af6d5e4aed12089719e8e4d77394c5409cf00
+    # via
+    #   -r https://raw.githubusercontent.com/mozilla-releng/scriptworker-scripts/4626000ccc5e264c69b6f5f06f67a7ed6362be14/notarization_poller/requirements/base.in
+    #   -r https://raw.githubusercontent.com/mozilla-releng/scriptworker-scripts/4626000ccc5e264c69b6f5f06f67a7ed6362be14/scriptworker_client/requirements/base.in
+    #   -r https://raw.githubusercontent.com/mozilla-releng/scriptworker/cd731fa4b02b10b430f9414df678f47640491931/requirements.txt
 jsmin==2.2.2 \
-    --hash=sha256:b6df99b2cd1c75d9d342e4335b535789b8da9107ec748212706ef7bbe5c2553b \
+    --hash=sha256:b6df99b2cd1c75d9d342e4335b535789b8da9107ec748212706ef7bbe5c2553b
     # via -r requirements.in
 json-e==4.4.1 \
-    --hash=sha256:7d9f6235f855ce70418b9d6158c043c588c3c3d7d0902972f3fb7c5399997ce9 \
-    # via -r https://raw.githubusercontent.com/mozilla-releng/scriptworker/a76a16da95e0d2483907b8a327b7a3f69743fbff/requirements.txt
+    --hash=sha256:7d9f6235f855ce70418b9d6158c043c588c3c3d7d0902972f3fb7c5399997ce9
+    # via -r https://raw.githubusercontent.com/mozilla-releng/scriptworker/cd731fa4b02b10b430f9414df678f47640491931/requirements.txt
 jsonschema==3.2.0 \
     --hash=sha256:4e5b3cf8216f577bee9ce139cbe72eca3ea4f292ec60928ff24758ce626cd163 \
-    --hash=sha256:c8a85b28d377cc7737e46e2d9f2b4f44ee3c0e1deac6bf46ddefc7187d30797a \
-    # via -r https://raw.githubusercontent.com/mozilla-releng/scriptworker-scripts/0ac27f13cc443cdadbaa715442ee720aec1da7a7/scriptworker_client/requirements/base.in, -r https://raw.githubusercontent.com/mozilla-releng/scriptworker/a76a16da95e0d2483907b8a327b7a3f69743fbff/requirements.txt
-jwcrypto==0.8 \
-    --hash=sha256:16e17faa4dce36551ade3a3ccb06236a61e5924ea1db163c9be9827acf935a82 \
-    --hash=sha256:b7fee2635bbefdf145399392f5be26ad54161c8271c66b5fe107b4b452f06c24 \
+    --hash=sha256:c8a85b28d377cc7737e46e2d9f2b4f44ee3c0e1deac6bf46ddefc7187d30797a
+    # via
+    #   -r https://raw.githubusercontent.com/mozilla-releng/scriptworker-scripts/4626000ccc5e264c69b6f5f06f67a7ed6362be14/scriptworker_client/requirements/base.in
+    #   -r https://raw.githubusercontent.com/mozilla-releng/scriptworker/cd731fa4b02b10b430f9414df678f47640491931/requirements.txt
+jwcrypto==1.0 \
```

Comment:
Nothing's jumping out at me: https://github.com/latchset/jwcrypto/releases/tag/v1.0.0

Reply:
Oops, I thought I stamped it at this point. Sorry!